hetmatpy/tests/test_pipeline.py

```diff
@@ -125,6 +125,21 @@ def test_calculate_sd(sum_of_squares, unsquared_sum, number_nonzero, expected_ou
       'beta': 2.0,
       'sum': 0.0
       }, 0.0),
+    # dwpc slightly larger than mean_nz, but within float error tolerance
+    ({'source_id': 'DB00193',
+      'target_id': 'DOID:0050425',
+      'source_name': 'Tramadol',
+      'target_name': 'restless legs syndrome',
+      'source_degree': 1,
+      'target_degree': 10,
+      'path_count': 1,
+      'dwpc': 7.323728709931218,
+      'n': 81600,
+      'nnz': 2086,
+      'n_perms': 200,
+      'mean_nz': 7.323728709931212,
+      'sd_nz': 0.0,
+      'p_value': 0.0}, 0.02556372549),
```

Comment:
Not sure we should pass the extra/wrong fields

Reply:
You're right, I just pushed a commit with the p_value field removed